src/kbmod_wf/task_impls/deep_plots.py

```diff
@@ -0,0 +1,418 @@
+        Keyword arguments passed forwards to `plt.figure`.
+    gs_kwargs : `dict` or `None`, optional
+        Keyword arguments passed forwards to `GridSpec`.
+    layout: `str`, optional
+        Figure layout is by default ``tight``.
+
+    Returns
+    -------
+    Figure : `obj`
+       Dataclass containing all of the created Axes, see `Figure`
+    """
+    fig_kwargs = {} if fig_kwargs is None else fig_kwargs
+    lk = fig_kwargs.pop("layout", None)
+    layout = "tight" if lk is None else lk
+    gs_kwargs = {} if gs_kwargs is None else gs_kwargs
+
+    fig = plt.figure(layout=layout, **fig_kwargs)
+
+    fig_gs = GridSpec(2, 2, figure=fig,  **gs_kwargs)
+    stamp_gs = gridspec.GridSpecFromSubplotSpec(1, 4, hspace=0.01, wspace=0.01, subplot_spec=fig_gs[1, 0])
+    stamp_gs2 = gridspec.GridSpecFromSubplotSpec(1, 4, hspace=0.01, wspace=0.01, subplot_spec=fig_gs[1, 1])
+
+    ax_left = fig.add_subplot(stamp_gs[:])
+    ax_left.axis('off')
+    ax_left.set_title('Coadded cutouts')
+
+    ax_right = fig.add_subplot(stamp_gs2[:])
+    ax_right.axis('off')
+    ax_right.set_title('Coadded cutouts normalized to mean values.')
+
+    stamps = np.array([fig.add_subplot(stamp_gs[i]) for i in range(4)])
+    
+    for ax in stamps[1:]:
+        ax.sharey(stamps[0])
+        plt.setp(ax.get_yticklabels(), visible=False)
+
+    normed = np.array([fig.add_subplot(stamp_gs2[i]) for i in range(4)])
+    for ax in normed[1:]:
+        ax.sharey(normed[0])
+        plt.setp(ax.get_yticklabels(), visible=False)
+
+    likelihood = fig.add_subplot(fig_gs[0, 0])
+    psiphi = likelihood.twinx()
+    likelihood.set_ylabel("Likelihood")
+    psiphi.set_ylabel("Psi, Phi value")
+    likelihood.set_xlabel("i-th image in stack")
+
+    sky = fig.add_subplot(fig_gs[0, 1], projection=wcs)
+    overlay = sky.get_coords_overlay('geocentricmeanecliptic')
+    overlay.grid(color='black', ls='dotted')
+    sky.coords[0].set_major_formatter('d.dd')
+    sky.coords[1].set_major_formatter('d.dd')
+
+    return Figure(fig, stamps, normed, likelihood, psiphi, sky)
+
+
+def result_to_skycoord(result, times, obs_valid, wcs):
+    """Return a collection of on-sky coordinates that match the result.
+
+    Take a result entry and return its SkyCoord positions on the sky.
+
+    Parameters
+    ----------
+    results : `Row`
+        Result
+    times : `np.array`
+        Array of MJD timestamps as floats.
+    obs_valid : `list[bool]`
+        A list of which observations are valid.
+    wcs : `WCS`
+        WCS
+
+    Returns
+    -------
+    coords : `SkyCoord`
+        World coordinates.
+    pos_valid : `list[bool]`
+        List of valid observations.
+    """
+    pos, pos_valid = [], []
+    times = Time(times, format="mjd")
+    dt = (times - times[0]).value
+
+    newx = result["x"]+dt*result["vx"]
+    newy = result["y"]+dt*result["vy"]
+    coord = wcs.pixel_to_world(newx, newy)
+    #pos.append(list(zip(coord.ra.deg, coord.dec.deg)))
```

Comment:
Did this ever get fixed? If not, and we still need it maybe replace with a longer comment about what was wrong